ksql-engine/src/main/java/io/confluent/ksql/codegen/CodeGenRunner.java

```diff
@@ -123,35 +121,39 @@ public ExpressionMetadata buildCodeGenFromParseTree(
       this.functionRegistry = functionRegistry;
     }
 
+    private void addParameter(Optional<Field> schemaField) {
+      parameters.add(new ParameterType(
+          SchemaUtil.getJavaType(schemaField.get().schema()),
```

Comment:
We should probably move the `if (schemaField.isPresent(...)` into this method

Reply:
I'll add `ifPresent()`.